From b4fd28098a74a2e7e876d76c2741a3f4910ba1a0 Mon Sep 17 00:00:00 2001
From: Benjamin Braddock <bbraddock@bbrad.dev>
Date: Sun, 17 Nov 2024 00:05:02 +0000
Subject: [PATCH] configs: starfive: increase SYS_XIMG_LEN

The decompression stage for the kernel was failing as the kernel was too
large for the predefined size.

Signed-off-by: Benjamin Braddock <bbraddock@bbrad.dev>
Upstream-Status: Pending
---
 configs/starfive_visionfive2_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/starfive_visionfive2_defconfig b/configs/starfive_visionfive2_defconfig
index 174ac24dc74..7ff9dbd565a 100644
--- a/configs/starfive_visionfive2_defconfig
+++ b/configs/starfive_visionfive2_defconfig
@@ -138,3 +138,4 @@ CONFIG_USB_KEYBOARD=y
 # CONFIG_WATCHDOG_AUTOSTART is not set
 CONFIG_WDT=y
 CONFIG_WDT_STARFIVE=y
+SYS_XIMG_LEN=0x2000000
-- 
2.47.0

